pkgs/top-level/all-packages.nix

```diff
@@ -19173,6 +19173,10 @@ with pkgs;
 
   ko = callPackage ../development/tools/ko { };
 
+  komac = callPackage ../by-name/ko/komac/package.nix {
+    inherit (darwin.apple_sdk_11_0.frameworks) SystemConfiguration;
```

Comment:
This is not needed, just using `darwin` would be okay

Reply:
You can directly use darwin as the input of derivation and do not need to expand it with callPackage.

Reply:
Sorry for multiple force push, I noticed it just after 22e8495e7fdd4c734a25dff317f0a171da2ee8ac 🙇 

Adding code only in by-name structure is much simple, thank you!